src/overlays/actors/ovl_En_Zoraegg/z_en_zoraegg.c

```diff
@@ -335,7 +335,7 @@ void func_80B32094(EnZoraegg* this, PlayState* play) {
 void func_80B320E0(EnZoraegg* this, PlayState* play) {
     if (Actor_HasParent(&this->actor, play)) {
         Flags_SetSwitch(play, ENZORAEGG_GET_FE00(&this->actor));
-        Actor_MarkForDeath(&this->actor);
+        Actor_Kill(&this->actor);
```

Comment:
Can replace with an early return.

Reply:
Does not match: 
![image](https://user-images.githubusercontent.com/12245827/195778032-09216e31-cc34-47b6-9040-9bfe3e005a69.png)